command/sub8_alarm/alarm_handlers/kill.py

```diff
@@ -8,13 +9,22 @@ class Kill(HandlerBase):
     def __init__(self):
         self._killed = False
         self._last_mission_killed = False
+        self.bagging_server = rospy.ServiceProxy('/online_bagger/dump', BaggerCommands)
 
     def raised(self, alarm):
         self._killed = True
+        self.bagger_dump()
 
     def cleared(self, alarm):
         self._killed = False
 
+    def bagger_dump(self):
+        """Call online_bagger/dump service"""
+        try:
+            bag_status = self.bagging_server(bag_name='kill_bag', bag_time=60)
```

Comment:
Hmmm, maybe it should only be the GNC ones and one camera / imaging sonar. Then again maybe we could spare 60 seconds of everything just for kill, not sure how large that gets.